tools/run_tests/helper_scripts/build_node.bat

```diff
@@ -38,5 +38,7 @@ for /f "delims=v" %%v in ('node --version') do (
   rmdir "%USERPROFILE%\.node-gyp\%%v\include\node\openssl" /S /Q
 )
 
+
+
 @rem rebuild, because it probably failed the first time
-call npm install --build-from-source
\ No newline at end of file
+call npm install --build-from-source %1 %2
```

Comment:
[%*](http://superuser.com/questions/149951/does-in-batch-file-mean-all-command-line-arguments) instead maybe ?